cfme/scripting/quickstart.py

```diff
@@ -163,12 +194,16 @@ def install_requirements(venv_path, quiet=False):
         current_packages = pip_json_list(venv_path)
         print("INFO:", REQUIREMENT_FILE, 'changed, updating virtualenv')
 
+    wheelhouse = get_wheelhouse(config_path)
+
     venv_call(
         venv_path,
         'pip', 'install',
         '-r', REQUIREMENT_FILE,
         '--no-binary', 'pycurl',
-        *(['-q'] if quiet else []))
+        ('-q' if quiet else ''),
```

Comment:
This I haven't tested yet, might have to play with the args/kwargs. python <3.5 doesn't support multiple *[] references in method calls, might have to combine -q and -f args logic here.